apps/backend/src/modules/rating/typedefs/rating.ts

```diff
@@ -0,0 +1,123 @@
+import { gql } from "graphql-tag";
+
+const typedef = gql`
+enum MetricName {
+    "Numerical metrics"
+    Usefulness
+    Difficulty
+    Workload
+
+    "Boolean metrics"
+    Attendance
+    Recording
+}
+
+enum Semester {
+    Fall
+    Spring
+    Summer
+    Winter
+}
+
+"""
+Ratings by class
+"""
+type AggregatedRatings {
+    "Class identifer"
+    subject: String!
+    courseNumber: String!
+    semester: Semester!
+    year: Int!
+    class: String!
```

Comment:
To keep naming consistent, I would use `number` or `classNumber` to represent a class here.